Source/Core/DolphinWX/Config/InterfaceConfigPane.cpp

```diff
@@ -9,6 +9,7 @@
 #include <wx/choice.h>
 #include <wx/language.h>
 #include <wx/sizer.h>
+#include <wx/gbsizer.h>
```

Comment:
Please sort this alphabetically,

Also consider putting up images of the differences before and after the change.